docs/20_smart-contracts/40_guides/10_resources_guide.md

```diff
@@ -0,0 +1,232 @@
+---
+title: Resources Guide
+---
+
+The EOS blockchain works with three system resources: CPU, NET and RAM. The EOS accounts need sufficient system resources to interact with the smart contracts deployed on the blockchain.
+
+* [RAM Resource](#ram-resource)
+* [CPU Resource](#cpu-resource)
+* [NET Resource](#net-resource)
+
+To allocate resources to an account use the `PowerUp Model` to [power up that account](#account-power-up).
```

Comment:
Because of the bullets above, this makes it seem like powerup is also where you get RAM